internal/pkg/v2/ui/common/routable/delegate.go

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright 2020 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package routable
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/application"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/ui/common"
+)
+
+// delegate contains references to Routable.
+type delegate struct {
+	routable common.Routable
+	execute  application.Executable
+}
+
+// NewDelegate is a factory function that returns a routable; fulfills the ui.Routable contract.
+func NewDelegate(routable common.Routable, execute application.Executable) *delegate {
+	return &delegate{
+		routable: routable,
+		execute:  execute,
+	}
+}
+
+// Execute defines the signature of an arbitrary executable unit of behavior.
```

Comment:
Why `defines the signature`? When read this I expect a type rather than a func.

Reply:
Comment was cut/paste from contract. Will update.